presto-spi/src/main/java/com/facebook/presto/spi/type/SqlTimestamp.java

```diff
@@ -66,8 +66,7 @@ public boolean equals(Object obj)
     @Override
     public String toString()
     {
-        SimpleDateFormat format = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss.SSS");
-        format.setTimeZone(getTimeZoneForKey(sessionTimeZoneKey));
-        return format.format(new Date(millisUtc));
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss.SSS");
+        return ZonedDateTime.ofInstant(Instant.ofEpochMilli(millisUtc), ZoneId.of(sessionTimeZoneKey.getId())).format(formatter);
```

Comment:
`Instant.ofEpochMilli(...).atZone(...)` instead of `ZonedDateTime.ofInstant(Instant.ofEpochMilli(...), ...)`